SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -466,6 +469,20 @@ void USpatialReceiver::ReceiveActor(Worker_EntityId EntityId)
 		}
 
 		EntityActor->UpdateOverlaps();
+
+		if (!NetDriver->IsServer())
+		{
+			if (RPCsOnEntityCreation* QueuedRPCs = StaticComponentView->GetComponentData<RPCsOnEntityCreation>(EntityId))
+			{
+				if (QueuedRPCs->HasRPCPayloadData())
+				{
+					ProcessQueuedActorRPCsOnEntityCreation(EntityActor, QueuedRPCs);
```

Comment:
My understanding was that we might not have authority over the client RPC endpoint yet. What's going to happen if client is expected to send a server RPC as a result of executing one of these queued RPCs?

Reply:
Moved the processing to HandleActorAuthority, when authority over ClientRPCEndpoint is gained.